Ginger/GingerCoreNET/RunLib/CLILib/CLIProcessor.cs

```diff
@@ -54,30 +55,30 @@ public void ExecuteArgs(string[] args)
                 args = newArgs;
             }
             
-            ParseArgs(args);                                        
+           await ParseArgs(args);                                        
         }
 
 
 
-        private void ParseArgs(string[] args)
+        private async Task ParseArgs(string[] args)
         {
             // FIXME: failing with exc of obj state
             // Do not show default version
             // Parser.Default.Settings.AutoVersion = false;            
-
-            int result = Parser.Default.ParseArguments<RunOptions, GridOptions, ConfigFileOptions, DynamicOptions, ScriptOptions, SCMOptions, VersionOptions, ExampleOptions, DoOptions>(args).MapResult(
-                    (RunOptions opts) => HandleRunOptions(opts),
-                    (GridOptions opts) => HanldeGridOption(opts),
-                    (ConfigFileOptions opts) => HandleFileOptions("config", opts.FileName, opts.VerboseLevel),
-                    (DynamicOptions opts) => HandleFileOptions("dynamic", opts.FileName, opts.VerboseLevel),
-                    (ScriptOptions opts) => HandleFileOptions("script", opts.FileName, opts.VerboseLevel),
-                    (SCMOptions opts) => HandleSCMOptions(opts),
-                    (VersionOptions opts) => HandleVersionOptions(opts),                        
-                    (ExampleOptions opts) => HandleExampleOptions(opts),
-                    (DoOptions opts) => HandleDoOptions(opts),
+            
+            int result = await Parser.Default.ParseArguments<RunOptions, GridOptions, ConfigFileOptions, DynamicOptions, ScriptOptions, SCMOptions, VersionOptions, ExampleOptions, DoOptions>(args).MapResult(
+                    async (RunOptions opts) => await HandleRunOptions(opts),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)